native/core/src/execution/jni_api.rs

```diff
@@ -233,11 +242,12 @@ pub unsafe extern "system" fn Java_org_apache_comet_Native_createPlan(
             stream: None,
             runtime,
             metrics,
+            metrics_update_interval,
+            metrics_last_update_time: Instant::now(),
```

Comment:
https://github.com/jedisct1/rust-coarsetime

@andygrove thoughts on a coarse time crate? The overhead on these clock_gettime() as used underneath Instant::now() can really add up. Maybe it's a premature optimization, but I also don't want a "death by 1000 cuts" scenario with gettime() all over the place.

Reply:
I ran `coarsetime`'s benchmark on my laptop:
```
coarsetime_now():          126.93 M/s
coarsetime_recent():       340.32 M/s
coarsetime_elapsed():      142.64 M/s
coarsetime_since_recent(): 340.34 M/s
stdlib_now():              51.37 M/s
stdlib_elapsed():          42.42 M/s
```
I'm a bit stunned that Rust's stdlib doesn't provide a nice way to get coarse time on its own, since the performance difference can be quite large and a lot of tasks don't need nanosecond precision.

Reply:
@mbutrovich I don't know much about coarse time, but I have no objection to adding this.

Reply:
I assume that we could make this change in a follow on PR

Reply:
I filed a follow on issue for this: https://github.com/apache/datafusion-comet/issues/1381